[2025-02-25T12:54:36.604+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.initialize_buckets manual__2025-02-25T12:54:36.014369+00:00 [queued]>
[2025-02-25T12:54:36.610+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.initialize_buckets manual__2025-02-25T12:54:36.014369+00:00 [queued]>
[2025-02-25T12:54:36.610+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-02-25T12:54:36.615+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): initialize_buckets> on 2025-02-25 12:54:36.014369+00:00
[2025-02-25T12:54:36.618+0000] {standard_task_runner.py:57} INFO - Started process 518 to run task
[2025-02-25T12:54:36.619+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'initialize_buckets', 'manual__2025-02-25T12:54:36.014369+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmpn31po00c']
[2025-02-25T12:54:36.623+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask initialize_buckets
[2025-02-25T12:54:36.632+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-25T12:54:36.648+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.initialize_buckets manual__2025-02-25T12:54:36.014369+00:00 [running]> on host 90c8125782a4
[2025-02-25T12:54:36.688+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='initialize_buckets' AIRFLOW_CTX_EXECUTION_DATE='2025-02-25T12:54:36.014369+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-25T12:54:36.014369+00:00'
[2025-02-25T12:54:36.881+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'raw' existe déjà.
[2025-02-25T12:54:36.881+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'staging' existe déjà.
[2025-02-25T12:54:36.881+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'curated' existe déjà.
[2025-02-25T12:54:36.882+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-25T12:54:36.887+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=initialize_buckets, execution_date=20250225T125436, start_date=20250225T125436, end_date=20250225T125436
[2025-02-25T12:54:36.932+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-25T12:54:36.946+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
